### PR TITLE
Lightning record picker multi value

### PR DESCRIPTION
**Pull request by arosenbach**
### What does this PR do?

New sample component to showcase `lightning-record-picker`: multi-value record picker

image

### What issues does this PR fix or reference?

#892

## The PR fulfills these requirements:

[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### CODE REVIEW

File: force-app/main/default/lwc/recordPickerMultiValue/recordPickerMultiValue.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            name: edge.node.Name.value
+        }));
+
+        // Add to selectedRecords
+        const selectedRecord = graphqlResults[0];
+        this.selectedRecords = [...this.selectedRecords, selectedRecord];
+
+        // We want the record picker input to be cleared
+        // each time the user selects a record suggestion
+        this._clearRecordPickerSeliction();
```
Comment (RamiNoufal) [inline force-app/main/default/lwc/recordPickerMultiValue/recordPickerMultiValue.js:112]: typo: _clearRecordPickerSelection

File: force-app/main/default/lwc/recordPickerMultiValue/recordPickerMultiValue.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <lightning-pill-container
+            items={pillItems}
+            onitemremove={handlePillRemove}
+        ></lightning-pill-container>
+        <template lwc:if={wireError}>
+            <c-error-panel errors={wireError}></c-error-panel>
+        </template>
+
+        <c-view-source source="lwc/recordPickerMultiValue" slot="footer">
+            Type a search terms and select records.
```
Comment (RamiNoufal) [inline force-app/main/default/lwc/recordPickerMultiValue/recordPickerMultiValue.html:27]: type: terms -> term

**Review by RamiNoufal — APPROVED** [review]

**Review by pozil — CHANGES_REQUESTED** [review]
Great work team. Thanks for the contribution.
Can you fix the bit that I suggested?

**Review by pozil — APPROVED** [review]

**Comment by pozil** [comment]
Great work. Merging now.

**Pull request merged**